assets/dev/scss/frontend/widgets/playing-cards.scss

```
@@ -0,0 +1,43 @@
+.elementor-playing-cards-wrapper {
+	--e-playing-card-red-color: red;
+	--e-playing-card-black-color: black;
+	--e-playing-card-box-shadow: 0px 0px 2px 0px rgba(0,0,0,0.55);
+
+	display: flex;
+	gap: .5rem;
+	align-items: center;
+
+	.elementor-playing-card {
```

We have switched from 'elementor' to 'e' inside class names.

```suggestion
	.e-playing-card {
```